src/lib_jit.c

```diff
@@ -141,6 +141,7 @@ LJLIB_CF(jit_attach)
   return 0;
 }
 
+#if LJ_HASJIT
```

Comment:
Oh, this is a good catch. Will you prepare a separate PR for this fix? So that we can merge this much sooner.

Reply:
@abhay1722 Have you missed this comment?

Reply:
@agentzh No, i took a note of it.
I'll be preparing a separate PR for the same, shortly afterwards.

Reply:
@abhay1722 Great. Thanks.

Reply:
@agentzh I have raised PR for the above changes, please refer to [this](https://github.com/openresty/luajit2/pull/51).

Reply:
Already merged.